### ISSUE TITLE
Components: `PostShare` block is missing a label for each connection

### ISSUE THREAD

**Issue by tyxla** [Automattic/wp-calypso#53002]
#### Steps to reproduce

1. Go to `/devdocs/blocks/post-share` after selecting a site in Calypso.
2. You'll notice that each connection lacks a label.

#### What I expected

Each connection should have a label so that would be more accessible.

#### What happened instead

Each connection is wrapped in divs instead.

#### Context / Source

Reported by @sarayourfriend in https://github.com/Automattic/wp-calypso/pull/52953#discussion_r634371685 while testing and reviewing #52953.

**Comment by tyxla** [comment]
Addressed by @nmertaydin in #53505. Thanks again for fixing it @nmertaydin 🙌
